pyfolio/tears.py

```diff
@@ -614,7 +614,8 @@ def create_returns_tear_sheet(returns, positions=None,
         live_start_date=live_start_date,
         ax=ax_return_quantiles)
 
-    if bootstrap and benchmark_rets is not None:
+    if ((bootstrap is not False)
```

Comment:
Does it work with this:
```suggestion
    if bootstrap and (benchmark_rets is not None)
```